src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala

```diff
@@ -35,6 +39,11 @@ class FFIDMetadataService(ffidMetadataRepository: FFIDMetadataRepository, matche
 
     val fileStatusRows = generateFileStatusRows(ffidMetadata)
 
+    logger.info("File check {} for fileId {} completed with status {}",
```

Comment:
How does this appear in the logs? It looks like there's some string interpolation going on, but do the values below end up in separate JSON fields as well for querying?

Reply:
Yep, you get the string as a message but you also get the fields included separately. This is an example that I got from running the tests.
```json
{
   "@timestamp":"2021-05-17T11:04:55.433+01:00",
   "@version":"1",
   "message":"File check FFID for fileId 07a3a4bd-0281-4a6d-a4c1-8fa3239e1313 completed with status Success",
   "logger_name":"FFIDMetadataService",
   "thread_name":"consignmentApi-akka.actor.default-dispatcher-6",
   "level":"INFO",
   "level_value":20000,
   "fileCheck":"FFID",
   "fileId":"07a3a4bd-0281-4a6d-a4c1-8fa3239e1313",
   "fileCheckStatus":"Success",
   "application":"consignment-api"
}
```

Reply:
That's really nice!